tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -957,6 +961,26 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
   LOG(INFO) << "Number of TensorRT candidate segments: "
             << initial_segments.size();
 
+  // Check if plugins can be aaccessed.
+  int num_trt_plugins = 0;
+  nvinfer1::IPluginCreator* const* trt_plugin_creator_list =
+    getPluginRegistry()->getPluginCreatorList(&num_trt_plugins);
```

Comment:
nit: please fix the indention and other formatting issues like line to long in the PR.